.circleci/config.yml

```diff
@@ -65,6 +65,12 @@ jobs:
             . venv/bin/activate
             dbt build -t snowflake --project-dir $DBT_PROJECT_DIR
 
+      - run:
+          name: "Run Tests - DuckDB"
+          command: |
+            . venv/bin/activate
+            dbt build -t duckdb --project_dir $DBT_PROJECT_DIR
```

Comment:
Hmmm, somehow CI failed on this line https://app.circleci.com/pipelines/github/calogica/dbt-date/83/workflows/f4415d81-0620-43a2-93e1-f0303c64f2f9/jobs/58?invite=true#step-108-27

Reply:
A silly typo-- fixed!